src/SlackReporter.ts

```diff
@@ -208,10 +217,20 @@ class SlackReporter implements Reporter {
       };
     }
 
-    if (!this.sendResults || this.slackChannels?.length === 0) {
```

Comment:
This condition was not true when `this.slackChannels` was undefined. 
With a change to slightly stricter validation I had to update the test data here 👉 https://github.com/ryanrosello-og/playwright-slack-report/pull/82/files#diff-62ed587fd8e65d151c73c06fa1f2fd6d4e3c45760c124f1505f41fa1d547f8cdR295-R322